src/app/views/collections/details/CollectionDetailsController.test.js

```diff
@@ -734,4 +740,56 @@ describe("When the component mounts with a collection id", () => {
         expect(collections.get.mock.calls.length).toBe(callsCounter);
         expect(component.state("drawerIsVisible")).toBe(false);
     });
+
+    it("and sec auth is enabled then get user email", () => {
+        const props = {
+            ...defaultProps,
+            collectionID: "test-collection-12345",
+            user: {
+                userType: "ADMIN",
+            },
+            isNewSignIn: true,
+        };
+
+        collections.get.mockImplementationOnce(() =>
+            Promise.resolve({
+                id: "test-collection-12345",
+                inProgress: [
+                    {
+                        description: {
+                            title: "Test",
+                            edition: "2024",
+                            language: "ENGLISH",
+                        },
+                        events: [
+                            {
+                                date: "2024-02-12T10:43:34.309Z",
+                                type: "EDITED",
+                                email: "test.user@email.com",
+                            },
+                        ],
+                    },
+                ],
+                complete: [],
+                reviewed: [],
+                approvalStatus: "NOT_STARTED",
+                events: [
+                    {
+                        date: "2024-02-12T10:43:10.803Z",
+                        type: "CREATED",
+                        email: "test.user@email.com",
+                    },
+                ],
+            })
+        );
```

Comment:
Not a review but this isn't working as you're anticipating - the mockcalls from earlier up are actually getting fired and it's not returning this when it gets the collection. 

Might want to try resetting the mocks between test runs or running them synchronously (I'm not sure if they're async atm) or using different mock data. 

It doesn't actually *solve* the problem per se, but it does get you further forward. Try more logs to see where the code is getting to and what's actually getting triggered. At the moment the getUserEmail function is *not* being called so the test is reporting correctly.